src/app/connections/list/list.component.html

```diff
@@ -63,39 +63,36 @@ <h4 class="modal-title pull-left">Confirm Delete</h4>
             <!-- Card Heading -->
 
             <div class="card-pf-heading-kebab">
-              <!--
-          
+
               <!-- Card Actions -->
 
-              <div class="dropdown pull-right dropdown-kebab-pf">
-                <button class="btn btn-link dropdown-toggle"
+              <div *ngIf="showKebab" dropdown class="dropdown dropdown-kebab-pf pull-right">
+                <button dropdownToggle class="btn btn-link dropdown-toggle"
                         type="button"
-                        id="dropdownKebabRight9"
-                        data-toggle="dropdown"
-                        aria-haspopup="true"
-                        aria-expanded="true">
+                        id="dropdownKebabRight9">
                   <span class="fa fa-ellipsis-v"></span>
                 </button>
-                <ul class="dropdown-menu dropdown-menu-right"
-                    aria-labelledby="dropdownKebabRight9">
-                  <li role="menuitem">
-                    <a [routerLink]="['/connections', connection.id]"
-                       class="dropdown-item">View</a>
-                  </li>
-                  <li role="menuitem">
-                    <a [routerLink]="['/connections/edit', connection.id]"
-                       class="dropdown-item">Edit</a>
-                  </li>
-                  <!--
-                  <li role="menuitem">
-                    <a class="dropdown-item">Duplicate</a>
-                  </li>
-                  -->
-                  <li>
-                    <a (click)="requestDelete(connection)"
-                       class="dropdown-item">Delete</a>
-                  </li>
-                </ul>
+                  <ul dropdownMenu class="dropdown-menu dropdown-menu-right" role="menu">
```

Comment:
@kahboom FYI this is the line that differs from the docs, just no `*` like in the docs for now...  We'll fix these templates then when we update to 1.6.6.  But anyways, we should use this for any dropdowns and not the patternfly jquery code.